app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,133 @@
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Keep track of your organization's resource ownership by mapping teams to Service Catalog services. | Add the Service Catalog service's owner when you [create a new Service Catalog service in {{site.konnect_short_name}}](https://cloud.konghq.com/service-catalog/create-service) |
+| Gain visibility into all your Services, including unrecognized or undiscovered APIs in your organization.  | [Service Catalog integrations](https://cloud.konghq.com/service-catalog/integrations) |
+| Consolidate key Service Catalog service health metrics, documentation, and API specs into a single dashboard, allowing you to interact with other tools from one place. | [Service Catalog dashboard](https://cloud.konghq.com/service-catalog/) |
+
+<!-- commenting this out until it's released:
+| Govern how services are created and maintained across your company to adhere to security, compliance, and engineering best practices. | Scorecards |-->
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Integration | These are applications, either {{site.konnect_short_name}}-internal or external, that act as sources from which you can ingest resources. For example, GitHub. |
+| Resource | An umbrella term that denotes entities ingested by Service Catalog from enabled integrations. A resource can range from an infrastructural component (like Gateway services, mesh services, databases, and caches) to an external application or tool (like code repositories, CI/CD infrastructure, and on-call systems) to a piece of documentation (like API specs). Resources can be mapped to one or more Services. |
+| Service Catalog service | A unit of software that is typically owned by a single team, exposes one or more APIs, and may be dependent on other Service Catalog services (as either upstream or downstream). A Service Catalog service can be thought of as a collection of one or more resources. |
+| Service Catalog | A comprehensive catalog of all resources and Service Catalog services running in your organization. |
+
+## FAQs
+
+<details><summary>How do Service Catalog services map to API products? What is the relationship there?</summary>
+
+{% capture service_mapping %}
+Service Catalog services do not directly map to API products. Rather, a Gateway service can be mapped to a Service Catalog service and you can then map the Gateway service to an API product version in Service Catalog.
+{% endcapture %}
+
+{{ service_mapping | markdownify }}
+
+</details>
+
+<details><summary>How do I view the health and status of my services in the Service Catalog?</summary>
+
+{% capture service_health %}
+Navigate to **Resources** in the Service Catalog and click on the Service Catalog service you want to view the health and status of.
+{% endcapture %}
+
+{{ service_health | markdownify }}
+
+</details>
+
+<details><summary>How can I identify and reuse existing resources?</summary>
```

Comment:
I'm not sure that I fully understand this question.

Perhaps the question should be rephrased into something like "How can I identify which resources are available for mapping to my Service Catalog services?"

And the answer would be something like: "Navigate to the **Resources** page in the Service Catalog to view a list of all available resources. These are ingested by Service Catalog from the integrations that have been installed and authorized, which are found in **Integrations** page.

The above is just a suggestion. Feel free to edit as you see fit!